src/lib.rs

```diff
@@ -40,8 +40,7 @@ impl Cifs {
             user: String::new(),
             password: String::new(),
             domain: "WORKGROUP".to_owned(),
-            workstation: std::env::var("HOSTNAME")
-                .unwrap_or("localhost".to_owned()),
+            workstation: "ANONYMOUS".to_owned(),
```

Comment:
is there any documentation that we can link to on why this is better? :sweat_smile:

Reply:
no, it is just because i am afraid that the hostname could be something that causes problems